dev/sample_mqttplugin/__init__.py

```diff
@@ -59,6 +59,10 @@ def __init__(self, sh):
         # Call init code of parent class (MqttPlugin)
         super().__init__()
 
+        # if you want to use an item to toggle plugin execution, enable the
+        # definition in plugin.yaml and uncomment the following line
+        #self._pause_item_path = self.get_parameter_value('pause_item')
```

Comment:
Should there be a check if the item exists..?

Reply:
Not necessarily; this never throws an error (but doesn't alert if the item is not present). As items are only initialized after plugin loading, testing for an existing item is not feasible here.